submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,105 @@
+
+[Visual of item groups: starter items, production items, oil processing items, rail system items, etc.]
+
+## How many machines should each recipe have?
+
+Mall assembling machines are *usually* idle more often than not because usually one does not build without interruption. Therefore ratios don't matter nearly as much as they usually do and often one machine per recipe is enough. 
+
+The exception to this rule is highly-used intermediates that would become serious bottlenecks. Namely, you would benefit from having a few assembling machines for electronic circuits and several machines for iron gear wheels (especially when making higher-tier transport belts). Meanwhile, for high-tech equipment and for modules, you need large quantities of expensive intermediate products such as advanced circuits, processing units, and low density structures. Therefore a mall for such items becomes difficult to supply using raw materials alone and it is worth building separate dedicated areas to craft and supply these expensive intermediates. 
+
+[visual idea?]
+
+## **How are the machine inputs supplied?**
+
+While getting resources to your mall area is one concern, another is how to feed them to your assembling machines. The first available method is handfeeding your assembling machines and their input chests. This makes it easy to supply to each machine with exactly as many ingredients as you want without overproducing. Due to their simplicity, handfed malls are also highly reprogrammable: You can set up a machine to make a one-time bulk order of a rarely needed item and then reassign the machine when you need something else. The disadvantages of handfed malls are that you have to do all the legwork yourself and that you have to wait for things to be crafted if you forget to supply a mall machine in advance.
+
+[Simple handfed mall example, with midgame items?]
+
+A more practical input supply method is using belts, which work very well at any stage of the game. Their supplying is continuous and you do not need input chests when using them. For a good starter mall, all you need to supply is a belt each for iron, copper, and steel; and you can make gears and circuits locally. Belt-based malls have no single design and building them is bit of a puzzle. If you carefully select which machines neighbor each other, you can build a decent compact early-game mall with only 3-4 belts. Alternatively, you can easily design your malls using mini-versions of the main bus technique, as presented [here]([Factorio Tutorials: Factory Shopping Mall - YouTube](https://www.youtube.com/watch?v=MPUcU37hXH4)) by KatherineOfSky and [here]([It is not a MALL; this is a Manufacturing HUB - Factorio 0.18 Tutorial/Guide/How-to - YouTube](https://www.youtube.com/watch?v=PLnv0O3cAnI)) by Nilaus.
```

Comment:
maybe you can include some hints from modded games, such as Space Exploration malls, or even AngelBob ones that can be insane to assemble? to highlight even more that on games with an increasing amount of intermediates and machines to use, it requires even more brain fu*****g to get something? ;)

Reply:
So you mean belt malls get extra difficult with several byproducts and intermediates being added to the game? Yeah a visual about that would be nice!